From: hololeap <hololeap@protonmail.com>
Signed-off-by: hololeap <hololeap@protonmail.com>

The `language-specifications` test suite tries to make connections to the
internet. It needs to be disabled.

diff -urN mustache-2.3.2/mustache.cabal mustache-2.3.2-r1/mustache.cabal
--- mustache-2.3.2/mustache.cabal	2022-01-20 13:27:20.348803958 -0700
+++ mustache-2.3.2-r1/mustache.cabal	2022-01-20 13:28:25.348803990 -0700
@@ -118,6 +118,7 @@
     , yaml
     , zlib
   default-language: Haskell2010
+  buildable: False
 
 test-suite unit-tests
   type: exitcode-stdio-1.0
